datahub-web-react/src/app/analytics/analytics.ts

```diff
@@ -45,6 +56,11 @@ export default {
         if (NODE_ENV === 'test') {
             return Promise.resolve();
         }
-        return analytics.track(eventTypeName, modifiedEvent, options, callback);
+        const trackingOptions = getMergedTrackingOptions(options);
+        return analytics.track(eventTypeName, modifiedEvent, trackingOptions, callback);
+    },
+    identify: (userId: string, traits?: any, options?: any, callback?: ((...params: any[]) => any) | undefined) => {
```

Comment:
Do we invoke identify method anywhere? I forget

Reply:
yup we invoke it in `checkAuthStatus`